test/jvm/src/main/scala/zio/test/reflect/Reflect.scala

```diff
@@ -0,0 +1,6 @@
+package zio.test.reflect
+
+object Reflect {
+  type EnableReflectiveInstantiation =
+    org.portablescala.reflect.annotation.EnableReflectiveInstantiation
+}
```

Comment:
This makes ZIO test have a dependency on portable reflection library. Hmm, we have to think if it's worth it or not. Does it support Dotty?